corehq/apps/case_importer/views.py

```diff
@@ -152,6 +159,10 @@ def _process_file_and_get_upload(uploaded_file_handle, request, domain, max_colu
             'you must first create an application with a case list.'
         ))
 
+    error_messages = custom_case_upload_file_operations(domain=domain, case_upload=case_upload)
+    if error_messages:
+        raise CustomImporterError(", ".join(error_messages))
```

Comment:
These would be the errors for the file. For now, this would be a single error only i.e the list would have only a single error but has been kept a list to add more errors/checks in future.